demos/projects/NXP/mimxrt1060/port/azure_iot_flash_platform.c

```diff
@@ -37,81 +40,160 @@ static AzureIoTResult_t prvBase64Decode( uint8_t * base64Encoded,
         return eAzureIoTErrorFailed;
     }
 
-    LogInfo( ( "Unencoded the base64 encoding\r\n" ) );
+    AZLogInfo( ( "Unencoded the base64 encoding\r\n" ) );
```

Comment:
too verbose?

Reply:
This matches the logging for the other ADU samples, so I'll leave it as is.